recipe/meta.yaml

```diff
@@ -32,6 +32,12 @@ requirements:
     - libpng >=1.6.28,<1.7
     - zlib 1.2.*
     - icu 58.*
+    - xorg-libx11 1.6.*
+    - xorg-libxext 1.3.*
+    - xorg-libxrender 0.9.*
+    - xorg-renderproto
+    - xorg-xextproto
+    - xorg-xproto
```

Comment:
Is this really needed on Windows? Cairo is working just fine on that OS without relying on xorg libraries.